website/src/styles/tokens-core.scss

```diff
@@ -69,19 +69,19 @@
   --colors-warning-900-warning: #261d0b;
   --colors-warning-base-warning: #ffcf0d;
   /* numbers */
-  --scale-none-scale: 0px;
-  --scale-50-scale: 4px;
-  --scale-100-scale: 8px;
-  --scale-150-scale: 12px;
-  --scale-200-scale: 16px;
-  --scale-300-scale: 20px;
-  --scale-400-scale: 24px;
-  --scale-500-scale: 28px;
-  --scale-600-scale: 32px;
-  --scale-700-scale: 40px;
-  --scale-800-scale: 48px;
-  --scale-900-scale: 48px;
-  --scale-1000-scale: 56px;
-  --scale-1200-scale: 64px;
-  --scale-1300-scale: 80px;
+  --scale-none-scale: 0rem;
```

Comment:
just `0` should be enough?

Reply:
these are copy pasted from figma, i don't think we should change too much in that, because it will be overwritten next time we update the tokens. and i don't think it's a very big problem...